typings/gapi.client.firestore-v1/index.d.ts

```diff
@@ -1,205 +1,163 @@
-// Type definitions for non-npm package Cloud Firestore API v1 1.0
+/* Type definitions for non-npm package Cloud Firestore API v1 0.0 */
```

Comment:
I generated a new set of definitions, and it looks like there's been more updates. Do you mind adding those in? (I did get some same `// Revision: 20220915` oddly enough).. and with that the Package.lock file also regenerated.

Reply:
Can you elaborate more on this? I am not sure I understand.

Reply:
I generated typings from the same library and it's been updated since you have added in these updates to @typings/gapi.client.firestore-v1.

If it helps I can add a PR to your fork really quick.

Reply:
Cool, you should have access to the branch, please do!